src/libraries/Microsoft.PowerFx.Core/Public/Values/OptionSetValue.cs

```diff
@@ -77,7 +77,7 @@ public string DisplayName
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
-            sb.Append($"{Type._type.OptionSetInfo.EntityName}.{Option}");
+            sb.Append($"{Type._type.OptionSetInfo.EntityName}.'{Option}'");
```

Comment:
I think there's a helper that will do this intelligently? Ie, only add if needed?

Reply:
This would also be incorrect if option contains ' since it wouldn't get properly escaped.